### PR TITLE
docs: Add demo link to ingest pages

### PR DESCRIPTION
**Pull request by kay-kim**
- Added as a tip for users who might want a bit more guidance on the ingest data pages.
   - Skipped the MySQL pages since those already require users get in touch to turn on the preview features.  But, can add if we think having the tip on those pages could also help.
- Added CSS for tips (not sure what the process is on updating CSS).

### CODE REVIEW

File: doc/user/content/get-started/quickstart.md [hunk 1]
```diff
@@ -306,6 +306,10 @@ DROP TABLE fraud_accounts;
 
 ## What's next?
 
+{{< note >}}
+For help getting started with your data or other questions about Materialize, you can schedule a [free guided trial](https://materialize.com/demo/?utm_campaign=General&utm_source=documentation).
```
Comment (JLDLaughlin) [inline doc/user/content/get-started/quickstart.md:310]: Nit, take it or leave it!

This sentence and the next seem to repeat themselves a bit the way they're currently structured. Maybe:

> To get started ingesting your own data from an external system like Kafka, MySQL or PostgreSQL, check the documentation for [sources](https://materialize.com/docs/sql/create-source/), and navigate to Data > Sources > New source in the [Materialize Console](https://console.materialize.com/) to create your first source.
>
> For help getting started with your data, or for other questions about Materialize, you can schedule a [free guided trial](https://materialize.com/demo/?utm_campaign=General&utm_source=documentation).

File: doc/user/content/ingest-data/redpanda.md [hunk 2]
```diff
@@ -15,7 +15,12 @@ menu:
 [//]: # "TODO(morsapaes) The Kafka guides need to be rewritten for consistency
 with the Postgres ones. We should include spill to disk in the guidance then."
 
-Because [Redpanda](https://vectorized.io/) is Kafka API-compatible, Materialize can process data from it in the same way it processes data from Kafka sources.
+Because [Redpanda](https://vectorized.io/) is Kafka API-compatible, Materialize
```
Comment (JLDLaughlin) [inline doc/user/content/ingest-data/redpanda.md:18]: I think this line break might be accidental?

**Review by JLDLaughlin — APPROVED** [review]
the changes look great to me!

the `?utm_campaign=General&utm_source=documentation` is a great add, and I'm excited to see what we learn. 👩🏻‍🍳

**Comment by morsapaes** [comment]
> Skipped the MySQL pages since those already require users get in touch to turn on the preview features. But, can add if we think having the tip on those pages could also help.

Better to, or it's likely we'll forget about it when we remove the PrPr banner.

> Added CSS for tips (not sure what the process is on updating CSS).

There...is none! For the guides, my only comment would be to put the annotation right below the page title, so it's immediately noticeable but less distracting once you started reading.

File: doc/user/content/ingest-data/amazon-msk.md [hunk 3]
```diff
@@ -16,6 +16,10 @@ with the Postgres ones. We should include spill to disk in the guidance then."
 
 This guide goes through the required steps to connect Materialize to an Amazon MSK cluster.
 
+{{< tip >}}
+For help getting started with your data or other questions about Materialize, you can schedule a [free guided trial](https://materialize.com/demo/?utm_campaign=General&utm_source=documentation).
```
[the remark this answers is not in the document]
Comment (morsapaes) [reply doc/user/content/ingest-data/amazon-msk.md:20]: Just a suggestion to shorten this up in the ones used in the ingestion guides. Also, so you don't have to copy-paste and edit this all over the place every time, you can create a [nested shortcode](https://gohugo.io/content-management/shortcodes/#nested-shortcodes).

**Review by morsapaes — COMMENTED** [review]
Thanks

**Pull request merged**